hydrus/data/crud.py

```diff
@@ -56,6 +56,10 @@ def insert(object_, id_=None):
     """Insert an object to database [POST] and returns the inserted object."""
     # NOTE: We are inserting the object, no need to check if similar one already exists.
     #       Data can be redundant/identical, they must have different "@id"
+
+    # TODO: @chrizandr You're not setting instance.type_ and you're not using keymap
+    # to dereference class types. Try any other category other then thermal for insertion
+    # Also, make sure crud.py passes all the tests in test_db.py (Currently all failing)

```

Comment:
@chrizandr aren't we using insert_classes.py to automatically insert classes to the Database. I think the work flow is we run `db_models.py` to setup database then we add classes using `insert_classes.py` and then we can insert data using `insert_data.py` which uses gemerator.py to automatically generated and insert data. Only the properties are automatically inserted during runtime. What say?